File: C/Circle/circle.c
/* This program takes the radius of a circle and prints the diameter, area,
   and circumference of the circle. */
#include <stdio.h>

int main( void )
{
    /* Variable declarations */
    int radius;

    /* Read value from user and store in variable radius */
    printf( "Enter radius: " ); /* Prompt user to enter value */
    scanf( "%d", &radius );

    /* Calcute and print diameter */
    printf( "Diameter is %f\n", radius * 2.0 );

    /* Calcute and print circumference */
    printf( "Circumference is %f\n", 2 * 3.14159 * radius );

    /* Calcute and print area */
    printf( "Area is %f\n", 3.14159 * radius * radius );

    return 0;
} /* end function */

File: Python/Circle/circle.py
#! /usr/bin/env python3
# This program takes a radius and returns the circumference, diameter, and area
# of the circle

radius = int( input( "Enter the radius: " ) ) # read radius from user

# Print the diameter
print( "The diameter is: {}".format( radius * 2 ) )

# Print the circumference
print( "The circumference is: {}".format( 2 * 3.14159 * radius ) )

# Print the area
print( "The area is: {}".format( 3.14159 * radius ** 2 ) )
